content/3-lifecycle/2-on-unmount/marko/Time.marko

```diff
@@ -0,0 +1,6 @@
+<let/time = new Date()/>
+<lifecycle() {
+  onMount () { this.timer = setInterval(_ => time = new Date()), 1000) }
+  onDestroy () { clearInterval(this.timer) }
+}/>
```

Comment:
```suggestion
<lifecycle
  onMount () { this.timer = setInterval(_ => time = new Date(), 1000) }
  onDestroy () { clearInterval(this.timer) }
/>
```

Reply:
Whoops, good catch — artifact from starting with `<effect() { … }/>` and then changing it to `<lifecycle>`